forte/data/caster.py

```diff
@@ -78,3 +78,43 @@ def input_pack_type():
     @staticmethod
     def output_pack_type():
         return MultiPack
+
+
+class DataPackBoxer(Caster[MultiPack, DataPack]):
+    """
+    This class creates a DataPack from a MultiPack, this DataPack is the only
+    contain of the original MultiPack, indexed by the :attr:`pack_name`.
```

Comment:
`:attr:pack_name` is a bit unclear. We should mention this comes from the configurtions.